xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.h

```diff
@@ -76,6 +77,8 @@ namespace OVERLAY {
     float m_y;
     float m_width;
     float m_height;
+    float m_source_width;
```

Comment:
Maybe document what those variables mean?

Reply:
try look if it is ok